src/main/java/org/apache/rocketmq/dashboard/service/impl/ConsumerServiceImpl.java

```diff
@@ -296,14 +296,22 @@ public boolean apply(MessageQueue o) {
 
     @Override
     public boolean deleteSubGroup(DeleteSubGroupRequest deleteSubGroupRequest) {
+        Set<String> brokerSet = this.fetchBrokerNameSetBySubscriptionGroup(deleteSubGroupRequest.getGroupName());
+        List<String> brokerList = deleteSubGroupRequest.getBrokerNameList();
+        boolean deleteInNsFlag = false;
+        // If the list of brokers passed in by the request contains the list of brokers that the consumer is in,
+        // delete RETRY and DLQ topic in namesrv
```

Comment:
//Delete... The first letter of the comment is capitalized and the style is consistent.

Reply:
done